extensions/wrapper/wrapper-broker-api/src/main/java/de/sovity/edc/ext/wrapper/api/broker/BrokerServerResource.java

```diff
@@ -43,4 +44,11 @@ public interface BrokerServerResource {
     @Produces(MediaType.APPLICATION_JSON)
     @Operation(description = "Query known Connectors")
     ConnectorPageResult connectorPage(ConnectorPageQuery query);
+
+    @POST
+    @Path("data-offer")
+    @Consumes(MediaType.APPLICATION_JSON)
+    @Produces(MediaType.APPLICATION_JSON)
+    @Operation(description = "Query data offer details")
+    DataOfferDetailResult dataOfferDetailPage(String connectorEndpoint, String assetId);
```

Comment:
refers to `PRIMARY KEY (connector_endpoint, asset_id),`

Reply:
Der unbeschriftete Parameter ist immer der Request Body.

Da wir aber zwei Parameter brauchen, bräuchten wir wie bei der Catalog Page eine "DataOfferPageQuery", die dann die zwei String-Felder beinhaltet